onnxruntime/core/mlas/inc/mlas.h

```diff
@@ -61,15 +61,15 @@ Module Name:
 #define MLAS_SUPPORTS_GEMM_DOUBLE
 #endif
 
-#if defined(MLAS_TARGET_AMD64_IX86) || defined(MLAS_TARGET_ARM64)
+#if defined(MLAS_TARGET_AMD64_IX86) || defined(MLAS_TARGET_ARM64) || (defined(MLAS_TARGET_ARM) && !defined(_MSC_VER))
```

Comment:
>_MSC_VER [](start = 107, length = 8)

Why _MSC_VER is excluded? It doesn't support certain instructions?

Reply:
There needs to be a different style of asm kernel ported to MSVC. This just gets in the Linux/Android piece.